pkg/test/tpm.go

```diff
@@ -775,3 +804,128 @@ func PCR0IsSet(txtAPI hwapi.APIInterfaces) (bool, error, error) {
 func checkTPM2NVAttr(mask, want, optional tpm2.NVAttr) bool {
 	return (1 >> mask & (want | optional)) == 0
 }
+
+func readPSLCPPolicy(txtAPI hwapi.APIInterfaces) (*tools.LCPPolicy, *tools.LCPPolicy2, error) {
```

Comment:
duplicate of PSIndexHasValidLCP, should use one common funtion to retrieve PSLCPPolicy.

Reply:
Done made it available for PSIndexHasValidLCP